firmware/firmware.ino

```diff
@@ -276,7 +276,17 @@ void setup()
 
 #if defined(DEBUG)
   Serial.begin(115200);
-#endif
+
+#ifdef SerialUSB
+  // all chips with "native" usb require waiting till `Serial` is true
+  // we wait at least 1500 msec, then go ahead anyway
+  unsigned int begin_time = millis();
+  while (! Serial && millis() - begin_time < 1500) {
+    delay(10);  // but at most 1.5 sec if not plugged in to usb
+  }
+#endif // SerialUSB
```

Comment:
see above.

Reply:
Specifically left the `while(!Serial)` because: it is only operative "if DEBUG", and on a SAMD board we'll lose the early Serial.print output.